src/app/posts/components/post-form/post-form.component.html

```diff
@@ -0,0 +1,22 @@
+<div data-target="#modal">
+    <form class="modal-body" [formGroup]="postForm" (ngSubmit)="onSubmit()">
+        <div class="postForm__item">
+            <input formControlName="author" class="postForm__item-input" type="text" placeholder="Article author"/>
+            <p class="validation" *ngIf="showValidationError('author')">Field author is required!</p>
+            <p class="validation" *ngIf="showValidationError('author', 'minlength')">Author field must be at least 4 characters long.</p>
+        </div>
+        <div class="postForm__item">
+            <input formControlName="title" class="postForm__item-input" type="text" placeholder="Article title"/>
+            <p class="validation" *ngIf="showValidationError('title')">Field title is required!</p>
+            <p class="validation" *ngIf="showValidationError('title', 'minlength')">Title field must be at least 5 characters long.</p>
```

Comment:
bad approach, add check error function to the class. You can store a Map<Error, ErrorDescription> and show the error label by using map[error]